adapters/gumgum/gumgum.go

```diff
@@ -127,25 +119,49 @@ func (g *GumGumAdapter) MakeBids(internalRequest *openrtb.BidRequest, externalRe
 	return bidResponse, errs
 }
 
-func preprocess(imp *openrtb.Imp) (string, error) {
+func preprocess(imp *openrtb.Imp) (*openrtb_ext.ExtImpGumGum, error) {
 	var bidderExt adapters.ExtImpBidder
 	if err := json.Unmarshal(imp.Ext, &bidderExt); err != nil {
 		err = &errortypes.BadInput{
 			Message: err.Error(),
 		}
-		return "", err
+		return nil, err
 	}
 
 	var gumgumExt openrtb_ext.ExtImpGumGum
 	if err := json.Unmarshal(bidderExt.Bidder, &gumgumExt); err != nil {
 		err = &errortypes.BadInput{
 			Message: err.Error(),
 		}
-		return "", err
+		return nil, err
+	}
+
+	if imp.Banner != nil {
+		bannerCopy := *imp.Banner
+		if bannerCopy.W == nil && bannerCopy.H == nil && len(bannerCopy.Format) > 0 {
+			format := bannerCopy.Format[0]
+			bannerCopy.W = &(format.W)
+			bannerCopy.H = &(format.H)
+		}
+		imp.Banner = &bannerCopy
+	} else if imp.Video != nil {
```

Comment:
Just want to verify, on a Banner+Video multiformat request, your bidder will ignore the video object and only bid banner bids?

Reply:
hi! apologies for the delay in responding! we currently don't support multiformat requests but I'll update this so we send both in case we support it in the future